content/docs/user-guide/data-management/index.md

```diff
@@ -0,0 +1,60 @@
+# Data Management with DVC
+
+DVC helps you manage and share arbitrarily large files, datasets, and ML models
+anywhere: mounted drives, Network Attached Storage (NAS), external devices, or
+remotely (SSH servers, cloud storage, etc.). Once the project is configured, you
+can manipulate files normally in your local workspace, and DVC tracks, restores,
+and synchronizes them across locations.
+
+![]() _Figure 1_
+
+## Separating data from code (local cache)
+
+First, large files, data artifacts, ML models are replaced with small
+[metafiles]; We call this process _codification_. The actual data files are then
+cached by DVC in a separate data store, and linked to your project.
+
+![]() _Figure 2_
```

Comment:
Figures are coming. They are meant to help paint a quick picture of the parts involved.

Reply:
The 2nd one specifically will probably have a before/after style contrasting ad hoc or loose data management vs. DVC's approach.